src/workerd/api/node/process.c++

```diff
@@ -92,9 +92,12 @@ namespace {
 
 jsg::JsObject ProcessModule::getVersions(jsg::Lock& js) const {
   auto versions = js.obj();
-  // Node.js version - represents the most current Node.js version supported
-  // by the platform, as defined in node-version.h
-  versions.set(js, "node"_kj, js.str(nodeVersion));
+
+  if (FeatureFlags::get(js).getPopulateProcessVersions()) {
```

Comment:
This only populates the node.js version, but the compat name refers to populating all process.versions?